src/mixins/raster-layer-line-mixin.js

```diff
@@ -212,6 +217,28 @@ function getScales({ size, color }, layerName, scaleDomainFields, xformDataSourc
     })
   }
 
+  if (typeof color === "object" && color.type === "density") {
```

Comment:
I haven't looked at the code that generates the accumulation scale for points/symbols, but I'm wondering if that code can be moved into a utility function. I'm thinking this code is pretty much the same.

Reply:
@vastcharade, I moved the getScales function to `utils-vega.js`. The logic in the function is almost the same for pointmap and linemap except one minor caveat that pointmap opacity slider controls rgb color opacity for points whereas linemap opacity is slider changes the opacity for the vega layer. We will probably change the opacity control for pointmap later.

Reply:
ok